src/main/java/com/carmen/app/entities/Car.java

```diff
@@ -1,21 +1,27 @@
 package com.carmen.app.entities;
 
 import java.io.Serializable;
-import java.util.Date;
+import java.time.LocalDateTime;
 import java.util.UUID;
 
+
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.NamedQuery;
 import javax.persistence.Table;
-import javax.persistence.Temporal;
-import javax.persistence.TemporalType;
+//import javax.persistence.Temporal;
+//import javax.persistence.TemporalType;
+
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
 
 
 @Entity
 @Table(name = "cars")
+@NamedQuery(name = "FindCars", query ="SELECT a FROM Car a")
```

Comment:
Usually the a named query is prefixed by the entity name, in this case the name would be "Car.FindCars", I know I know it will work without the prefix, but is a good practice to have, as we may know simply but looking at the query usage where this query was defined.